ompi/mca/common/ompio/common_ompio_aggregators.c

```diff
@@ -896,11 +896,14 @@ int mca_common_ompio_split_initial_groups(ompio_file_t *fh,
     int size_smallest_group = 0;
     int num_groups = 0;
     int ret = OMPI_SUCCESS;
+    OMPI_MPI_COUNT_TYPE bytes_per_agg_group = 0;
```

Comment:
Is `OMPI_MPI_COUNT_TYPE` the right type for a byte count here or should it be `MPI_Aint`?